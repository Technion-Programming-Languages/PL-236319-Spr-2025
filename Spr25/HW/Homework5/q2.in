use_module(library(yall)).
print("===TEST START===").

once(fib(X,7)).
once(fib(12,8)).

once(padovan(X,1)).
once(padovan(X,7)).

once(catalan(X,1)).
once(catalan(X,7)).

once(multisum(2,X,Y,Z)).
once(multisum(8,X,Y,Z)).




