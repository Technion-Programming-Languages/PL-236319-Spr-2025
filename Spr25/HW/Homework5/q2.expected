true.

X = 8.

false.

X = 1.

X = 4.

X = 1.

X = 429.

X = 1,
Y = 2,
Z = 3.

X = 33,
Y = 19,
Z = 2055.


